### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Ev-1**

### CODE REVIEW

File: cogs/music/basic_commands.py [hunk 1]
```diff
@@ -439,3 +439,19 @@ async def _forceplay(self, ctx, *, query: str):
 
     if not player.is_playing:
         await player.play()
+
+
+@commands.command(name='forcedisconnect')
+@checks.dj_or(alone=True)
+async def _forcedisconnect(self, ctx):
+    """ Searches and plays a song from a given query. """
```
Comment (Roxedus) [inline cogs/music/basic_commands.py]: Copy-pasta 😋

File: cogs/music/basic_commands.py [hunk 2]
```diff
@@ -439,3 +439,19 @@ async def _forceplay(self, ctx, *, query: str):
...
+    try:
+        player = self.bot.lavalink.player_manager.get(ctx.guild.id)
+        player.queue.clear()
+        await player.stop()
+    except Exception:
+        self.logger.exception("Error forcedisconnecting")
```
Comment (Roxedus) [inline cogs/music/basic_commands.py:453]: Probably worth logging the actual exception too.

Comment (Ev-1) [reply cogs/music/basic_commands.py:453]: Seems
https://www.loggly.com/blog/exceptional-logging-of-exceptions-in-python/

File: cogs/music/basic_commands.py (hunk 1 above)
Comment (Ev-1) [reply cogs/music/basic_commands.py]: fixed

**Pull request merged**